GitUI/UserControls/RevisionGrid/FormRevisionFilter.Designer.cs

```diff
@@ -470,6 +483,7 @@ private void InitializeComponent()
         private System.Windows.Forms.Label _NO_TRANSLATE_lblBranches;
         private System.Windows.Forms.TextBox BranchFilter;
         private System.Windows.Forms.CheckBox CurrentBranchOnlyCheck;
+        private System.Windows.Forms.Label _NO_TRANSLATE_lblCurrentBranchOnlyCheck;
```

Comment:
How is this new Label used? At least the `Before` screenshot is not update.

Reply:
Before, `CurrentBranchOnlyCheck` was in column 2 with the text label. I had not removed that label, pushed an update.
Since _Show current branch only_ is dominating `BranchFilter` the checkbox was shifted to column 1 with no space for the text label.
`_NO_TRANSLATE_lblCurrentBranchOnlyCheck` should have been added to column 0, but is too long why it was added to column 2 instead.

Before screenshot updated.

Reflog should be added to FormFilter as well as the branch filter as that option dominates the other filters. This PR cleans up the logic.
![image](https://user-images.githubusercontent.com/6248932/188276487-e6ba31be-adff-48b7-83d3-8363aaadb9ce.png)